src/main/kotlin/no/nav/klage/kaka/domain/Saksdata.kt

```diff
@@ -174,6 +174,10 @@ class Saksdata(
             validationErrors.add(
                 createMustBeFilledValidationError(SaksdataView::mottattKlageinstans.name)
             )
+        } else if (LocalDate.now().isBefore(mottattKlageinstans)) {
```

Comment:
Tror jeg ville snudd denne. Sjekket om `mottattKlageinstans` var etter dagens dato. Men dette kan jo være veldig subjektivt :) Hvis det er naturlig for deg slik det er skrevet nå så bare behold :)